doc/script_commands.txt

```diff
@@ -5874,6 +5873,47 @@ Examples:
 // Enables the current char to move again.
 	pcblockmove getcharid(3),0;
 
+---------------------------------------
+
+*pcblock <type>,<option>{,<account ID>};
+
+Prevents the player from doing the following actions when the <option> is
+true(1), and false(0) will allow those actions again. If there is no
+account ID given, the script will run for the current attached player.
+
+The <type> listed are :-
+	BLOCK_MOVE     0x01
+	BLOCK_ATTACK   0x02
+	BLOCK_SKILL    0x04
+	BLOCK_USEITEM  0x08
+	BLOCK_CHAT     0x10
+	BLOCK_IMMUNE   0x20
+	BLOCK_SITSTAND 0x40
+	BLOCK_COMMAND  0x80
+
+Examples:
+
+// Make the current attached player invulnerable, same as @monsterignore
+	pcblock BLOCK_IMMUNE, true;
+
+// Prevents the current char from attacking and using skills
+	pcblock BLOCK_ATTACK|BLOCK_SKILL, true, getcharid(3);
+
+---------------------------------------
+
+*getpcblock({<account ID>})
+
+This command will return the options of the current attached player used by
+'pcblock' command. Additionally, you may insert account ID within the bracket
+to check on another player.
+
+Examples:
+
+	if (getpcblock() & BLOCK_IMMUNE)
+		mes "You are invulnerable!";
+
+	if (getpcblock() & (BLOCK_ATTACK|BLOCK_SKILL))
+		mes "You are unable to attack and use any skills.";
```

Comment:
Nitpick (because otherwise potentially misleading for a reader): this isn't accurate, the "and" should be an "or", since this condition is true when the player is unable to attack OR use any skills.